app/mailers/admin_mailer.rb

```diff
@@ -24,4 +24,24 @@ def submission(submission:, user:, artist:)
       subject: "Submission ##{@submission.id}"
     ) { |format| format.html { render layout: "mailer_no_footer" } }
   end
+
+  def artwork_updated(submission:, user_data:, artwork_data:, changes:)
+    assigned_admin = AdminUser.find_by(gravity_user_id: submission.assigned_to)
```

Comment:
Should assigned cataloguer be emailed too?

Reply:
We might want to return early or send to a fallback address if no assignee is found.